### PR TITLE
check specific op (lstm, gru) in converted graph

### PR DESCRIPTION
**Pull request by lucienwang1009**
In unittests, check if converted graph contains specific ops, such as LSTM, GRU.

Other changes:
1. add time major and forget bias unittests for LSTM.
2. remove the requirement for forget bias in lstm_rewriter.

### CODE REVIEW

**Comment by nbcsm** [comment]
as we discussed, let's find a better way instead of ```must_contain_op="GRU", must_contain_op_num=2```

File: tf2onnx/rewriter/lstm_rewriter.py [hunk 1]
```diff
@@ -73,7 +73,7 @@ def get_weight_and_bias(self, match):
         if not ft:
             return None
 
-        if not (ft.value == 1 and self.g.get_dtype(b_e.output[0]) == self.g.get_dtype(ft_bias.output[0])):
+        if not self.g.get_dtype(b_e.output[0]) == self.g.get_dtype(ft_bias.output[0]):
```
Comment (pengwa) [inline tf2onnx/rewriter/lstm_rewriter.py]: maybe we should check ft_bias being a scalar?

**Comment by lucienwang1009** [comment]
> as we discussed, let's find a better way instead of `must_contain_op="GRU", must_contain_op_num=2`

as we talked offline, a graph validator that checks graph attributes will be passed to run_test_case.

File: tf2onnx/rewriter/lstm_rewriter.py (hunk 1 above)
Comment (lucienwang1009) [reply tf2onnx/rewriter/lstm_rewriter.py]: Sure! using `np.ndim(ft.value) == 0` to check whether ft is a scalar

File: tests/test_gru.py [hunk 2]
```diff
@@ -47,7 +47,8 @@ def test_single_dynamic_gru(self):
         input_names_with_port = ["input_1:0"]
         feed_dict = {"input_1:0": x_val}
         output_names_with_port = ["output:0", "cell_state:0"]
-        self.run_test_case(feed_dict, input_names_with_port, output_names_with_port, rtol=1e-03, atol=1e-06)
+        self.run_test_case(feed_dict, input_names_with_port, output_names_with_port, rtol=1e-03, atol=1e-06,
+                           graph_validator=lambda g: len(group_nodes_by_type(g)["GRU"]) == 1)
```
Comment (nbcsm) [inline tests/test_gru.py]: you may want to define a common helper function, e.g. ```check_op_count(g, op_type, expected_count)```

Comment (nbcsm) [reply tests/test_gru.py]: and maybe also define```check_gru_count```, ```check_lstm_count``` in corresponding files to save some typing.

**Pull request merged**